### PR TITLE
[RFC] Formally track aspiring approvers as reviewers

### PR DESCRIPTION
**Pull request by rkoster**
Extend the working group yaml defenitions to optionally include a `reviewers` key with a list of contributors who like to become approvers.
The github org automation should use this information to create a github team for the area with `-reviewers` suffix and __read__ permissions.

This will later allow automatic assignment of PRs to be reviewed, which will help these contributors more easily statisfy the requirments to become approvers.

This PR formalizes the pattern introduced in: https://github.com/cloudfoundry/community/pull/324

### CODE REVIEW

**Review by stephanme — CHANGES_REQUESTED** [review]
We need to mention or extend toc/rfc/rfc-0008-role-change-process.md, i.e. how would a contributor gain reviewers role.

I guess the process will be similar as for promotion to contributor:
- pre-requisite: user is contributor and files a PR
- 2 existing approvers review the PR with +1
- WG lead will merge or close

**Comment by pivotal-marcela-campo** [comment]
I do think it is useful to help contributors get approver status. It provides an standardised way of assigning reviews and following up on that within the PR to make sure the review gets done before merging.

**Comment by stephanme** [comment]
A use case discussed in CAPI Open Office Hour: Reviewers role can also be used to grant read access to Concourse pipelines of a WG area that the team doesn't want to make publicly accessible (e.g. due to risk of revealing credentials in the build logs).

**Comment by stephanme** [comment]
Q: Should reviewers be part of the team wg-[WORKING-GROUP-NAME]?

https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0005-github-teams-and-access.md as of today would exclude them (membership: All approvers and leads for a WG). The intended use case for the WG global team (for organization and tagging) suggests to add them. There are no repo permissions granted by this team.

**Comment by ameowlia** [comment]
Yes, the ARP working group would like to have reviewer teams. This way I can automatically assign PRs to these teams.

File: toc/rfc/rfc-formal-aspiring-approvers-as-reviewers.md
Comment (rkoster) [reply toc/rfc/rfc-formal-aspiring-approvers-as-reviewers.md:12]: I would go optionally since otherwise, we would have to update all other working groups, even the ones who won't use this feature.

**Comment by rkoster** [comment]
> Q: Should reviewers be part of the team wg-[WORKING-GROUP-NAME]?
> 
> https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0005-github-teams-and-access.md as of today would exclude them (membership: All approvers and leads for a WG). The intended use case for the WG global team (for organization and tagging) suggests to add them. There are no repo permissions granted by this team.

**Review by stephanme — APPROVED** [review]

**Comment by emalm** [comment]
TOC voted today to start the final comment period on this RFC, concluding August 2.

**Pull request merged**
